### PR TITLE
SC-5644 - footer

### PR DESCRIPTION
**Pull request by haniapalac**
# Description

## Links to Tickets or other pull requests

https://ticketsystem.schul-cloud.org/browse/SC-5644
## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)
https://docs.schul-cloud.org/display/CAP/Acceptance+Criteria

### CODE REVIEW

File: theme/brb/views/lib/footer.hbs [hunk 1]
```diff
@@ -1,25 +1,37 @@
-<footer class="hide-inline small-footer" id="footer">
+<footer id="footer" class="hide-inline small-footer" tabindex="0">
 	<div class="imprint">
 		<div class="center">
-			<img src="/images/footer-logo.png" alt="Anbieterlogo" />
+			<img src="/images/footer-logo.png" alt="" role="presentation"/>
```
Comment (luizzaly) [inline theme/brb/views/lib/footer.hbs:4]: why did you remove the alt here?

Comment (haniapalac) [reply theme/brb/views/lib/footer.hbs:4]: This is not a link and looked like it is only for presentational purposes?

Comment (dominikbruechner) [reply theme/brb/views/lib/footer.hbs:4]: 👍
